secrets.test.sample

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+#: Mail settings
+export MAIL_SERVER=''
+export MAIL_PORT=''
+export MAIL_USE_SSL=True
+export MAIL_USE_TLS=False
+export MAIL_DEFAULT_SENDER=''
+export MAIL_USERNAME=''
+export MAIL_PASSWORD=''
+export SITE_SUPPORT_EMAIL=''
+export ADMINS=['']
+export ADMIN_NUMBERS=['']
+
+#: Keys for tests specifically
+export FACEBOOK_OAUTH_TOKEN=''
+
+#: Twitter integration
+export OAUTH_TWITTER_KEY=''
+export OAUTH_TWITTER_SECRET=''
+
+#: GitHub integration
+export OAUTH_GITHUB_KEY=''
+export OAUTH_GITHUB_SECRET=''
+
+#: Google integration
+export OAUTH_GOOGLE_KEY=''
+export OAUTH_GOOGLE_SECRET=''
+
+#: Recaptcha for the registration form
+export RECAPTCHA_PUBLIC_KEY=''
+export RECAPTCHA_PRIVATE_KEY=''
+
+#: Google Analytics
+export GA_CODE=''
+
+#: Typekit code
+export TYPEKIT_CODE=''
```

Comment:
Leave the environment variable setup to the end user, since travis will have to be configured manually anyway.